lib/helper.rb

```diff
@@ -0,0 +1,21 @@
+def position_taken?(board, idx)
+  if board[idx] == 'X' || board[idx] == 'O'
+    true
+  else
+    false
+  end
+end
+
+def valid_move?(board, position)
+  idx = position - 1
+  if idx.between?(0, 8) && !position_taken?(board, idx)
+    true
+  else
+    false
+  end
+end
```

Comment:
Their best place, to respect the OOP principle, must in `lib/board.rb`, that is in `GameBoard` class.
**Why?**: only because, they act on `board array`. Without the board array, they don't have the reason to exists